### PR TITLE
Add support to delete commit comment reaction

### PR DESCRIPTION
**Pull request by wesleimp**
Delete a commit comment reaction
- `DELETE /repos/:owner/:repo/comments/:comment_id/reactions/:reaction_id`
- Docs: https://developer.github.com/v3/reactions/#delete-a-commit-comment-reaction

### CODE REVIEW

File: github/reactions.go [hunk 1]
```diff
@@ -109,6 +110,23 @@ func (s ReactionsService) CreateCommentReaction(ctx context.Context, owner, repo
 	return m, resp, nil
 }
 
+// DeleteCommentReaction delete the reaction for a commit comment.
```
Comment (gmlewis) [inline github/reactions.go]: s/delete/deletes/

File: github/reactions.go [hunk 2]
```diff
@@ -109,6 +110,23 @@ func (s ReactionsService) CreateCommentReaction(ctx context.Context, owner, repo
...
+//
+// GitHub API docs: https://developer.github.com/v3/reactions/#delete-a-commit-comment-reaction
+func (s *ReactionsService) DeleteCommentReaction(ctx context.Context, owner, repo string, id, reactionID int64) (*Response, error) {
```
Comment (gmlewis) [inline github/reactions.go]: Let's please change `id` to `commentID` here.

**Comment by wesleimp** [comment]
> Thank you, @wesleimp!
> 
> I think we should also add a new `DeleteCommentReactionByRepoID` method in this same PR which supports the comment in the docs that says:
> 
> > Note: You can also specify a repository by repository_id using the route DELETE /repositories/:repository_id/comments/:comment_id/reactions/:reaction_id.
> 
> You can either call out to a private method where these two public methods just generate the URL and pass it to the private method that does the rest (which I personally prefer), or just have to fully complete public methods. Your choice.

@gmlewis I think now the changes are ok

**Comment by wesleimp** [comment]
@googlebot I signed it!

**Comment by gmlewis** [comment]
@martinssipenko - if you feel like reviewing and approving, your help would be greatly appreciated!

**Comment by gmlewis** [comment]
Thank you, @martinssipenko !
Merging.

**Comment by gmlewis** [comment]
Relates to: #1442.

**Pull request merged**